web/README.md

```diff
@@ -103,9 +103,10 @@ In this project, you may run the following commands to build all packages:
     -   This will build the wasm binary and every node package (notably selfhosted and extension).
     -   Output will be available in the `dist/` folder of each package (for example, `./packages/selfhosted/dist`).
     -   You may also use `npm run build:debug` to disable Webpack optimizations and activate the (extremely verbose) ActionScript debugging output.
-    -   There is `npm run build:dual-wasm` as well, to build a second WebAssembly module that makes use of some WebAssembly extensions,
-        potentially resulting in better performance in browsers that support them, at the expense of longer build time.
+    -   There is `npm run build:dual-wasm` as well, to build a second WebAssembly module that makes use of only MVP WebAssembly extensions,
+        potentially resulting in support for more browsers, at the expense of longer build time.
     -   `npm run build:repro` enables reproducible builds. Note that this also requires a `version_seal.json`, which is not provided in the normal Git repository - only specially-marked reproducible source archives. Running this without a version seal will generate one based on the current state of your environment.
+    -   Since we use `RUSTC_BOOTSTRAP` in order to use certain Nightly flags with stable Rust when building dual-wasm, you will also need to run `rustup component add rust-src` with either of the previous two commands.
```

Comment:
It might be worth mentioning that this is only needed for `build:dual-wasm` now.

Reply:
I do say "with either of the previous two commands"

What would you suggest?

Reply:
Right, I didn't see enough context as for what the previous two commands actually are (I'm on mobile ATM), sorry.

Not nitpicking, and maybe I'm not right, but the direct reason for needing `rust-src` is that we do `build-std`. The `RUSTC_BOOTSRAP` envvar just happens to be necessary for it, since it's not stable yet.